backend/oasst_backend/api/v1/users.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+import datetime
+from uuid import UUID
+
+from fastapi import APIRouter, Depends, Query
+from oasst_backend.api import deps
+from oasst_backend.models import ApiClient
+from oasst_backend.prompt_repository import PromptRepository
+from oasst_shared.schemas import protocol
+from sqlmodel import Session
+from starlette.responses import Response
+from starlette.status import HTTP_200_OK
+
+router = APIRouter()
+
+
+@router.get("/{user_id}/messages")
+def query_user_messages(
+    user_id: UUID,
+    api_client_id: UUID = None,
+    max_count: int = Query(10, gt=0, le=25),
```

Comment:
I think we allow a higher value here, e.g. `le=1000` ...

Reply:
Changed to 1000